.github/workflows/lock.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ github.token }}
+          issue-lock-inactive-days: '30'
+          issue-exclude-labels: 'Discussion'
+          issue-lock-comment: >
+            This issue has been automatically locked since there
+            has not been any recent activity after it was closed.
+            Please open a new issue for related bugs.
```

Comment:
Should we include the SO/discord link?

Something about linking back to relevant issues?

Reply:
I think make sense. I’ve noticed that it’s hard to find SO/Discord links unless going to online doc and scroll to footer. It’s better to make those links are more visible.